File: pset1/mario more/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
/* Asking user for a height */
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    /* The while rule */
    while (n < 1 || n > 8);
    /* Number of rows */
    for (int i = 0; i < n; i++)
    {
        /* Number of left spaces */
        for (int x = i; x < n - 1; x++)
        {
            printf(" ");
            
        }
        /* Number of left hashtags */
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        /* Number of middle spaces */
        for (int s = 0; s < 2; s++)
        {
            printf(" ");
        }
        /* Number of right hashtags */
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");    
    }
    
}

File: pset6/mario/more/mario.py
from cs50 import get_int
# While loop
while True:
    n = get_int("Height: ")
    
    if n > 0 and 9 > n:
        # Number of rows
        for i in range(n):
            # Number of spaces
            for s in range(n-1): 
                print(" ", end="")
            # Number of #
            for j in range(i + 1): 
                print("#", end="")
            for s in range(2):
                print(" ", end="")
            for j in range(i + 1): 
                print("#", end="")
            # Decreasing
            n = n - 1 
            print()
        exit(0)